sdk/src/rest/commands/utils/versions.ts

```diff
@@ -0,0 +1,68 @@
+import type { DirectusVersion } from '../../../schema/version.js';
+import type { UnpackList } from '../../../types/index.js';
+import type { RestCommand } from '../../types.js';
+
+/**
+ * Compare an existing version with the main item.
+ *
+ * @param id Primary key of the version.
+ *
+ * @returns All fields with different values, along with the hash of the main item and the information whether the current
+version is outdated (main item has been updated since the creation of the current version)
+ */
+export const compareVersion =
```

Comment:
I feel the naming here is too vague and possibly even misleading. lets use a more feature specific and descriptive name here for these SDK functions like `compareDraftVersion` or `compareRevisionVersion`.

Reply:
Makes sense, `compareContentVersion` sounds fitting with Content Versioning 👍 In that case, I'll proceed with updating all of them such as `readVersion`/`updateVersion` to `readContentVersion`/`updateContentVersion` respectively as well.

Reply:
resolved via https://github.com/directus/directus/pull/20000/commits/e71199bf9555eb164c374ddbecd433d81ad6de15 🙌